### ISSUE TITLE
Suggestion: Avatar dialog and better avatar controls

### ISSUE THREAD

**Issue by Tarun80** [mumble-voip/mumble#2159]
I've been thinking lately about setting better avatar controls for Mumble.

Essentially, a dialog window that pulls a few (new) set values from the murmur.ini file. That is, avatar width, height, max file size, and supported image types. A bit like how some forum software handles their avatars. Letting people know the requirements.

**Comment by hacst** [comment]
Just to make sure I understand the FR: This would be a client side feature to let people optimize their avatars?

**Comment by Tarun80** [comment]
Yes. Essentially clicking Self > Change Avatar would display this dialog, populating fields pulled from the murmur.ini file

File Location: [Textbox] [Browse...]
Maximum dimensions: 185x185
Supported types: jpg, png, bmp, gif
Maximum file size: 512KB
